docs/mp/health/01_introduction.adoc

```diff
@@ -36,6 +36,17 @@ include::{common-deps-page-prefix-inc}[tag=maven-dependency]
         </dependency>
 ----
 
+To enable built-in health checks add the following dependency
+(or us the <<mp/introduction/02_microprofile.adoc, helidon-microprofile bundle>> )
+
+[source,xml]
+----
+        <dependency>
+            <groupId>io.helidon.microprofile.health</groupId>
```

Comment:
The `groupId` is `io.helidon.health` (no `microprofile`).

Reply:
Fixed